app/api/helpers/mail.py

```diff
@@ -73,6 +74,22 @@ def send_email(to, action, subject, html):
     return True
 
 
+def send_email_with_action(user, action, **kwargs):
+    """
+    A general email helper to use in the APIs
+    :param user: email or user to which email is to be sent
+    :param action:
+    :param kwargs:
+    :return:
+    """
+    send_email(
+        to=user,
```

Comment:
it always needs to be email, right? What do you mean by "or user to which email is to be sent"? I think naming email is better.